app/tests/Unit/App/Helper/KompetenzenHelperTest.php

```diff
@@ -17,14 +18,16 @@
             "max",
             $kompetenzen,
             $requiredKompetenzen,
-            'icon-component'
+            'icon-component',
+            CategoryId::BILDUNG_UND_KARRIERE
         );
 
-        expect($result)->toHaveCount(2)
-            ->and($result[0]->drawEmpty)->toBeTrue()
-            ->and($result[0]->drawHalfEmpty)->toBeFalse()
-            ->and($result[1]->drawEmpty)->toBeTrue()
-            ->and($result[1]->drawHalfEmpty)->toBeFalse();
+        expect($result->kompetenzen)->toHaveCount(2)
+            ->and($result->kompetenzen[0]->drawEmpty)->toBeTrue()
+            ->and($result->kompetenzen[0]->drawHalfEmpty)->toBeFalse()
+            ->and($result->kompetenzen[1]->drawEmpty)->toBeTrue()
+            ->and($result->kompetenzen[1]->drawHalfEmpty)->toBeFalse()
+            ->and($result->ariaLabel)->toBe('Deine Kompetenzen im Bereich Bildung & Karriere: 0 von 2');
```

Comment:
```suggestion
            ->and($result->ariaLabel)->toBe('Deine Kompetenzsteine im Bereich Bildung & Karriere: 0 von 2');
```

Nochmal abklären vom Wording, was die hier gern hätten.